[NetApp] Update to 2022-09-01 add tests (#34629)
Azure/azure-sdk-for-net: sdk/netapp/Azure.ResourceManager.NetApp/CHANGELOG.md

## 1.1.0-beta.1 (Unreleased)

### Features Added

> - Added BackupRestoreFiles to NetAppVolumeResource NetAppVolumeBackupResource to rstore the specified files from the specified backup to the active filesystem

Reviewer: typos
```suggestion
- Added BackupRestoreFiles to NetAppVolumeResource NetAppVolumeBackupResource to restore the specified files from the specified backup to the active file system
```

- Added BreakFileLocks to NetAppVolumeResource to allow clients to break file locks on a volume
- Added FileAccessLog to NetAppVolumeData, a lag indicating whether file access logs are enabled for the volume, based on active diagnostic settings present on the volume.
- Added propperty PreferredServersForLdapClient to NetAppAccountActiveDirectory, a comma separated list of IPv4 addresses of preferred servers for LDAP client
- VolumeRelocation to NetAppVolumeDataProtection

> - DataStoreResourceId to NetAppVolumeGroupVolume

Reviewer: what do these two lines mean? Changed VolumeRelocation to NetAppVolumeDataProtection?

Author: added

> - Added Tags to NetAppVolumeQuotaRulePatch and AddTag to NetAppVolumeQuotaResource

Reviewer: ```suggestion
- Added `Tags` property to NetAppVolumeQuotaRulePatch and `AddTag`, `RemoveTag` and `SetTags` to NetAppVolumeQuotaResource
```

- Added RestoreFiles for NetAppVolumeBackupResource

### Breaking Changes

> - NetAppVault is no longer needed scheduled for deprication

Reviewer: here is a typo
```suggestion
- NetAppVault is no longer needed scheduled for deprecation
```

> - Plan to depricate vaultId from NetAppVolumeBackupConfiguration

Reviewer: ```suggestion
- Plan to deprecate vaultId from NetAppVolumeBackupConfiguration
```
